### DISCUSSION TITLE
I can not download TGM plugin activation

### DISCUSSION THREAD

**Discussion by HARUN400** [community/community#108618] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I can not download TGM plugin activation

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @HARUN400! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
